neuralop/models/uno.py

```diff
@@ -139,12 +146,25 @@ def __init__(self,
         self.preactivation = preactivation
         self._incremental_n_modes = incremental_n_modes
 
-        assert len(self.layer_configs) == n_layers
+        
+
+        self.layer_configs = []
+        for l in range(n_layers):
+            l_config = {}
+            l_config['out_channels'] = uno_out_channels[l]
+            l_config['n_modes'] = uno_n_modes[l]
+            l_config['res_scaling'] = uno_scalings[l]
+            self.layer_configs.append(l_config)
```

Comment:
We don't need to create this `layer_configs` dict, let's directly store the `uno_foo`